frontend/packages/knative-plugin/src/topology/components/side-panel/knative-connectors-tab-sections.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { Edge } from '@patternfly/react-topology';
+import i18next from 'i18next';
+import TopologyEdgeResourcesPanel from '@console/topology/src/components/side-bar/TopologyEdgeResourcesPanel';
+import {
+  TYPE_EVENT_SOURCE_LINK,
+  TYPE_KAFKA_CONNECTION_LINK,
+  TYPE_REVISION_TRAFFIC,
+} from '../../const';
+
+const connectorTypeToTitle = (type: string): string => {
+  switch (type) {
+    case TYPE_REVISION_TRAFFIC:
+      return i18next.t('knative-plugin~Traffic distribution connector');
+    case TYPE_EVENT_SOURCE_LINK:
+      return i18next.t('knative-plugin~Event source connector');
+    case TYPE_KAFKA_CONNECTION_LINK:
+      return i18next.t('knative-plugin~Kafka connector');
```

Comment:
I would suggest adding these as the label on edge data model while forming the model data transformers. Then we wouldn't need to create resource link sections for the type. 

element.getLabel() is being used incase resource-link is not found in SideBarHeader component